assets/src/js/menu_editor.js

```diff
@@ -157,7 +157,7 @@ const menuEditorRestrictions = () => {
    * @return string The current location slug
    */
   const getCurrentLocation = () => {
-    return locations.map(location => {
+    return locations.filter(location => {
```

Comment:
@lithrel I had to change the `map` to `filter`. Bacause `map` will always returns a value and in consequence if you try to get the zero index of the returned array the value will be always `null` in cases where you have more than 1 item into the list.
i.e. Having this array: `['navigation-bar-menu', 'donate-menu']`, then:

If you want to check `navigation-bar-menu` the result will be:
```
['navigation-bar-menu', null] // Zero index = 'navigation-bar-menu'
```
If you want to check `donate-menu` the result will be:
```
[null, 'donate-menu'] // Zero index = null
```

So, it will never get the config of the `donate-menu`

Reply:
Yes I saw that a bit late, the fix was also part of the code I had left there https://github.com/greenpeace/planet4-master-theme/pull/1604#discussion_r810962793, works for me :+1: